frontend/taipy-gui/src/components/Taipy/Chat.tsx

```diff
@@ -424,6 +463,7 @@ const Chat = (props: ChatProps) => {
                                 senderId={senderId}
                                 message={`${row[columns[1]]}`}
                                 name={columns[2] ? `${row[columns[2]]}` : "Unknown"}
+                                image={row[columns[3]] ? `${row[columns[3]]}` : null}
```

Comment:
image={columns[3] && columns[3] != "_tp_index" ? `${row[columns[3]]}` : undefined}

Reply:
```
image={columns[3] && columns[3] != "_tp_index" ? `${row[columns[3]]}` : undefined}
```

I'm not sure how this logic works yet, but when I used this. The messages without image create `<img src="null">` element. 

![Screenshot from 2024-10-18 08-59-35](https://github.com/user-attachments/assets/402513df-31cb-4962-af32-fda0a48102a2)

I use this for now.
```
image={columns[3] && columns[3] != "_tp_index" && row[columns[3]] ? `${row[columns[3]]}` : undefined}
```

Reply:
way better, tx